src/wp-includes/media.php

```diff
@@ -2103,13 +2219,30 @@ function wp_iframe_tag_add_loading_attr( $iframe, $context ) {
 
 	// Get loading attribute value to use. This must occur before the conditional check below so that even iframes that
 	// are ineligible for being lazy-loaded are considered.
-	$value = wp_get_loading_attr_default( $context );
+	$optimization_attrs = wp_get_loading_optimization_attributes(
+		'iframe',
+		array(
+			/**
+			 * The concrete values for width and height are not important here for now
+			 * since fetchpriority is not yet supported for iframes.
+			 * TODO: Use WP_HTML_Tag_Processor to extract actual values once support is
+			 * added.
+			 */
+			'width'   => false === strpos( $iframe, ' width="' ) ? null : 100,
```

Comment:
```suggestion
			'width'   => str_contains( $iframe, ' width="' ) ? null : 100,
```

Reply:
Thanks, your suggestion is only missing the `!`, otherwise it's doing the opposite. Will commit.